classes/controller/ModuleFrontController.php

```diff
@@ -92,7 +92,7 @@ public function initContent()
      */
     protected function l($string, $specific = false, $class = null, $addslashes = false, $htmlentities = true)
     {
-        if (isset($this->module) && is_a($this->module, 'Module')) {
+        if (is_a($this->module, 'Module')) {
```

Comment:
Optional suggestion:
```suggestion
        if ($this->module instanceof Module::class)) {
```

Reply:
Fixed